NUnitAPITests/TestSettings.json

```diff
@@ -4,9 +4,7 @@
     "key": "",
     "baseUrl": "https://www.pivotaltracker.com/services/v5"
   },
-  "trello": {
-    "token": "",
-    "key": "",
-    "baseUrl": "https://api.trello.com/1"
-  }
+    "trello": {
+        "token": "",
+        "baseUrl": "https://api.trello.com/1/"
```

Comment:
The `key` value is missing, you delete it and now the `TestSettings.json` is invalid. Please fix it.